### PR TITLE
Add gunpowder recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Depends on:
- https://github.com/conda-forge/staged-recipes/pull/19532

### CODE REVIEW

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Comment by jakirkham** [comment]
cc @funkey (for awareness)

**Comment by jaimergp** [comment]
Thanks John! We'll wait a few days in case the author wants to join as a maintainer here :)

**Comment by funkey** [comment]
Oh, fantastic, I wasn't aware of those efforts! Thank you :)

Just a small thing, we should probably change the description of Gunpowder to "A library to facilitate machine learning on large, multi-dimensional images." (instead of the Greentea reference that we grew out of long ago).

I updated the `setup.py` in `master` accordingly.

**Comment by goanpeca** [comment]
@funkey do you want to be added as a maintainer?

**Comment by funkey** [comment]
Not necessarily (I assume that means to maintain the conda package), but thanks for the offer!

**Comment by goanpeca** [comment]
This one is ready for review https://github.com/orgs/conda-forge/teams/staged-recipes :)

File: recipes/gunpowder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - gunpowder
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/funkey/gunpowder
+  summary: Data loading DAG for Greentea.
```
Comment (jaimergp) [inline recipes/gunpowder/meta.yaml:42]:
```suggestion
  summary: A library to facilitate machine learning on large, multi-dimensional images..
```

**Pull request merged**